lms/djangoapps/lti_provider/tests/test_signature_validator.py

```diff
@@ -117,4 +117,4 @@ def test_verification_parameters(self, verify_mock):
         headers = {'Content-Type': content_type}
         SignatureValidator(self.lti_consumer).verify(request)
         verify_mock.assert_called_once_with(
-            request.build_absolute_uri(), 'POST', body, headers)
+            request.build_absolute_uri(), 'POST', body.decode('utf-8'), headers)
```

Comment:
Can you just make the `body` string above be a unicode string instead?  That seems like a simpler solution if it will work.

Reply:
sorry there was an error in solution.